src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -306,37 +305,36 @@ private void readResources(Document document) {
         String inputDescription = getInputMessageDescription(grpcServiceName, methodname);
         Message input = new Message(requestName, false, false, inputDescription);
         String httpOptionPath = method.flatPath();
-        List<String> methodSignatureParams = new ArrayList<>();
-
-        for (Schema requiredParam : method.requiredParams().values()) {
-          Field requiredField = schemaToField(requiredParam);
-          Option fieldBehaviorOption = new Option("google.api.field_behavior");
-          fieldBehaviorOption.getProperties().put("", "REQUIRED");
-          requiredField.getOptions().add(fieldBehaviorOption);
-          input.getFields().add(requiredField);
-          methodSignatureParams.add(requiredField.getName());
+        Map<String, String> methodSignatureParamNames = new LinkedHashMap<>();
```

Comment:
I would find a description of the key, value helpful here. I see below key is the identifier and the value is the name, but describing it here would be helpful. I assume the former is fully qualified and the latter is not?

Reply:
I added a comment describing what is going on there.